test/e2e/lib/pages/login-page.js

```diff
@@ -24,10 +24,19 @@ export default class LoginPage extends AsyncBaseContainer {
 		const driver = this.driver;
 		const userNameSelector = By.css( '#usernameOrEmail' );
 		const passwordSelector = By.css( '#password' );
+		const changeAccountSelector = By.css( '#loginAsAnotherUser' );
 
 		await driverHelper.waitTillPresentAndDisplayed( driver, userNameSelector );
+
 		// await driverHelper.waitTillFocused( driver, userNameSelector );
-		await driverHelper.setWhenSettable( driver, userNameSelector, username );
+		try {
+			// If there is already a logged in user, we display the Gravatar and 'Continue as [user]' button and hide login form
+			await driverHelper.setWhenSettable( driver, userNameSelector, username );
+		} catch {
```

Comment:
Lets move this wait in to the try below. If the userNameSelector isn't there, the test will fail.